content/en/docs/configuring-sdk/endpoints.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Configuring Client Endpoints"
+linkTitle: "Endpoints"
+date: "2020-11-12"
+description: Customizing service client endpoints.
+---
+
+In most cases you use the endpoint that is pre-configured for a service. However, you can specify a custom endpoint,
+such as for pre-release versions of the service.
+
+A [EndpointResolver]({{< apiref "aws#EndpointResolver" >}}) can be configured to provide custom endpoint resolution
+logic for service clients. A custom endpoint resolver has the ability to override the resolution of a service's endpoint
+resolution for all endpoints, or a just specific regional endpoint. Custom endpoint resolver have the ability trigger
+the service's endpoint resolution logic to fallback if a custom resolver does not wish to resolve a requested
+endpoint. [EndpointResolverFunc]({{< apiref "aws#EndpointResolverFunc" >}}) can be used to easily wrap functions to
+satisfy the `EndpointResolver` interface.
+
+A `EndpointResolver` can be easily configured by passing the resolver wrapped with
+[WithEndpointResolver]({{< apiref "aws#WithEndpointResolver" >}}) to
+[LoadDefaultConfig]({{< apiref "config#LoadDefaultConfig" >}}), allowing for the ability to override endpoints when
+loading credentials, as well as configuring the resulting `aws.Config` with your custom endpoint resolver.
```

Comment:
i think the `WithEndpointResolver` is a `config` package item not `aws` in the `apiref` link.

Reply:
Correct, had this one mixed up with EndpointResolverFunc